panel/README.md

```diff
@@ -2,19 +2,15 @@
 
 ## Installation
 
-The Panel dev server is a proxy of another web server like Apache or Nginx that serves an actual Kirby project. Point the dev server to that project with an `.env` file similar to the `.env.example` one in this folder:
+Our setup expects that you are running [Kirby Sandbox](https://github.com/getkirby/sandbox), our our local test environment, at `http://sandbox.test`. You can reach the Panel at `http://sandbox.test`.
 
-```
-VUE_APP_DEV_SERVER = http://kir.by
-```
-
-…where `kir.by` is the project domain. You can set up such a domain by editing your machine's `hosts` file and your web server's configuration.
+If you are using a different setup, you might need to create a `/panel/vite.config.custom.js` where you can point `vite` to the right server.
 
-Make sure to add a dummy CSRF token in the `site/config/config.php` of your Kirby project:
+When developing, make sure to put Kirby into development mode by adding the following lien to  `site/config/config.php` of your Kirby project:
 
 ```php
 return [
-  'api.csrf' => 'dev'
+  'panel.dev' => true
```

Comment:
This is already handled by the sandbox, so it's not needed to add this manually.

Reply:
Since the README also addresses setups not using the sandbox, I think we need to include this anyways.